chaincode/tuple.go

```diff
@@ -163,6 +163,44 @@ func queryModels(db *LedgerDB, args []string) (outModels []outputModel, err erro
 	return
 }
 
+func queryModelPermissions(db *LedgerDB, args []string) (outputPermissions, error) {
+	var out outputPermissions
+	inp := inputKey{}
+	err := AssetFromJSON(args, &inp)
+	if err != nil {
+		return out, err
+	}
```

Comment:
Further more I think we should use an `InputHash` here not an `inputKey` since it's indeed a hash. @Kelvin-M do you think we could update the backend in that direction?
I'll add `modelHash` for now.

Reply:
Yes we can, but the modification must be done also for all object to  have the same usage. So let's do it later because it will be too long to verify that everything works :)

Reply:
I was speaking about this smart contract in particular since `queryModelPermissions` returns the Permission of the Model matching the `Hash` in the arguments.